docs/generated/packages/cypress/executors/cypress.json

```diff
@@ -73,11 +76,13 @@
       },
       "baseUrl": {
         "type": "string",
-        "description": "The address (with the port) which your application is running on."
+        "description": "The address (with the port) which your application is running on.",
+        "x-priority": "important"
       },
       "browser": {
         "type": "string",
-        "description": "The browser to run tests in."
+        "description": "The browser to run tests in.",
+        "x-priority": "important"
```

Comment:
I obviously have no idea but are you sure this is something folks will set manually most of the time? To me it seems like you would always want to use the same browser so this could be set in nx.json and forgotten about in many cases